ebs/ebs.go

```diff
@@ -802,8 +802,9 @@ func (d *Driver) DeleteVolume(req Request) error {
 		"DetachedAt":   time.Now().String(),
 	}
 	err := d.UpdateTags(volume.EBSID, detachTags)
+	// Do not fail when adding tracking tags, this should not be critical
 	if err != nil {
-		return err
+		log.WithField("volume-id", id).Warnf("failed adding detach tracking tags to volume: %v", err)
```

Comment:
If we returned here after a successful detach we would not reach the return statement (which removes the object from convoy's state), this could cause issues where convoy thinks it has a volume and someone else already claimed it because the volume was available in ebs.